ocient/manifest.json

```
@@ -0,0 +1,52 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "c2769280-0bee-4e1a-a6ea-0c3e66fe1500",
+  "app_id": "ocient",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Collect performance metrics and monitor overall Ocient cluster health",
+    "title": "Ocient",
+    "media": [],
```

Please reference the image of your dashboard here.
```suggestion
    "media": [
        "media_type": "image",
        "caption": "[insert caption here]",
        "image_url": "images/ocient_dashboard.png"
    ],
```